### ISSUE TITLE
Making feedback improvements to enhance "perceived performance" during operations

### ISSUE THREAD

**Issue by winslet** [hyperledger/composer#1969]
We would like to look into adding small microinteractions throughout the playground to increase the perceived performance of operations.

Some ideas:

- Submit transactions in the background and show notification when they are done, allowing users to continue working whilst they happen.
- When deploying to HSBN for the first time, could we provide more information on what is going on (other than a spinner). 

The bottom line - make waiting enjoyable :)

**Comment by sstone1** [comment]
> The bottom line - make waiting enjoyable :)
Add animated goat videos alongside the spinner?
